tasks/src/Zero-shot-classification/data.ts

```diff
@@ -0,0 +1,30 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		},
+	],
+	
+	models: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "i",
+		},
+	],
+	summary:      "Zero-shot classification is a machine learning classification task where the training data does not contain any labels for the classes that the model will be asked to predict. The model must learn to map the input data to the correct class based solely on the input data. This is in contrast to traditional classification tasks, where the model is trained on data that is labeled with the correct class.",
```

Comment:
Can you simplify the language here a little for a broader audience 🙂

Reply:
I will try to simplify but will be adding zero-shot-text-classification work here?

Reply:
@merveenoyan can you clear the doubts I mentioned above !!